gammapy/utils/fitting/parameter.py

```diff
@@ -354,10 +362,10 @@ def from_dict(cls, val):
                 Parameter(
                     name=par["name"],
                     factor=float(par["value"]),
-                    unit=par["unit"],
-                    min=float(par["min"]),
-                    max=float(par["max"]),
-                    frozen=par["frozen"],
+                    unit=par["unit"] if "unit" in par.keys() else '',
```

Comment:
Is this a dict? Then you can use `par.get("unit", "")` pattern for defaults.

Reply:
Yes